int64 totalSidesIn
---
int64 totalSidesOut
